x-pack/platform/plugins/shared/cases/server/saved_object_types/cases/versioning.test.ts

```diff
@@ -73,4 +73,20 @@ describe('caseSavedObjectType model version transformations', () => {
       });
     });
   });
+
+  describe('Model version 4 to 5', () => {
+    const version4Fields = ['incremental_id.text'];
```

Comment:
The saved objects migrator will not inject `incremental_id.text` into your fields because those are ES mappings that should not affect your SO attributes (document source). If it did a lot of things would break!